central/blob/datastore/store/store.go

```diff
@@ -46,6 +51,15 @@ func wrapRollback(ctx context.Context, tx *pgPkg.Tx, err error) error {
 
 // Upsert adds a blob to the database
 func (s *storeImpl) Upsert(ctx context.Context, obj *storage.Blob, reader io.Reader) error {
+	if err := sac.VerifyAuthzOK(scopeChecker.WriteAllowed(ctx)); err != nil {
+		return err
+	}
+	// Augment permission because we require read permission internally
+	ctx = sac.WithGlobalAccessScopeChecker(ctx,
```

Comment:
I'm not sure why we need the augmentation.  We have to have write to upsert and as far as I know, our SAC choices are `Access_NO_ACCESS`, `Access_READ_ACCESS`, and `Access_READ_WRITE_ACCESS`.  Only `Access_READ_WRITE_ACCESS` gives us write and it also has read.  If it doesn't we may have other issues with SAC.  Unless we need this augmentation for the scope key.  And then I wonder if we are allowing more than we should.

Reply:
That was what I thought, but you may run the test by removing the product side of changes and keep the test side of changes and run it. Some comments also pointing that. Access_READ_WRITE_ACCESS means write permission.

Reply:
In that case, it might be worth adding the SAC team to review.

Reply:
Right, @rhybrillou It looks like Access_READ_WRITE_ACCESS is not intuitive to its name. Do you know any background of it?